### PR TITLE
Don't reboot a device if non-daemon mode

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

**Comment by mike-sul** [comment]
@mike-scott @doanac This is a cowboy/hot fix to not reboot a device if aklite runs in a non-daemon mode.
I feel like I would like to refactor code around that fix/place but it will take some time, so I'll do it along the way while adding new functionality.

File: src/main.cc [hunk 1]
```diff
@@ -351,7 +351,9 @@ int main(int argc, char* argv[]) {
       LOG_DEBUG << "Running " << (*cmd_to_run).first;
       LiteClient client(config);
       ret_val = (*cmd_to_run).second(client, commandline_map);
-      is_reboot_required = client.isRebootRequired();
+      if (cmd == "daemon") {
+        is_reboot_required = client.isRebootRequired();
+      }
```
Comment (doanac) [inline src/main.cc:356]: maybe add an "else { you need to reboot now }"?

Comment (mike-sul) [reply src/main.cc:356]: A similar message is yielded before this place https://github.com/foundriesio/aktualizr-lite/blob/ead98c90887c71aa15b4993036b52f671149ce12/src/helpers.cc#L530

Comment (doanac) [reply src/main.cc:356]: ah - okay. looks good then. lets ship it

**Pull request merged**